src/server/lock_manager/mod.rs

```diff
@@ -56,7 +57,7 @@ pub struct LockManager {
     waiter_count: Arc<AtomicUsize>,
 
     /// Record transactions which have sent requests to detect deadlock.
-    detected: Arc<Vec<Mutex<HashSet<TimeStamp>>>>,
+    detected: Arc<Vec<CachePadded<Mutex<HashSet<TimeStamp>>>>>,
```

Comment:
The document says `the wrapped data is never dropped`. Is the inner content dropped manually?

Reply:
Will be dropped in `crossbeam = "0.7.2"`. I guess you referred to an old version.